src/index.html

```diff
@@ -1,13 +1,220 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="/" class="logo">
+              <img
+                src="./images/logo.svg"
+                class="logo__image"
+                alt="The MET"
+              >
+            </a>
+
+            <div class="header__icon-container">
+              <a href="tel:+1 212-535-7710" class="icon icon--phone"></a>
+
+              <a href="#menu" class="icon icon--menu-opener"></a>
+            </div>
+          </div>
+          <h1 class="header__title">
+            Welcome
+            <br>
+            to The MET
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
```

Comment:
Again you can rename it menu__container. Create a placeholder or a mixin for container and use it that way so all of the style is same between block__container